go.mod

```diff
@@ -16,59 +16,61 @@ require (
 	github.com/onsi/ginkgo v1.16.4 // indirect
 	github.com/onsi/gomega v1.16.0 // indirect
 	github.com/otiai10/copy v1.2.0
-	github.com/spf13/cobra v1.4.0
-	github.com/stretchr/testify v1.7.2
-	golang.org/x/crypto v0.1.0
-	golang.org/x/oauth2 v0.0.0-20220524215830-622c5d57e401
-	golang.org/x/tools v0.1.12
-	k8s.io/apimachinery v0.17.2
+	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
+	github.com/spf13/cobra v1.6.1
+	github.com/stretchr/testify v1.8.1
+	golang.org/x/crypto v0.3.0
+	golang.org/x/oauth2 v0.1.0
+	golang.org/x/tools v0.6.0
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	k8s.io/apimachinery v0.24.3
 	k8s.io/code-generator v0.17.2
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
 )
 
 require (
-	github.com/google/go-containerregistry v0.10.0
-	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
+	github.com/google/go-containerregistry v0.13.0
```

Comment:
nit: Do you mind moving all the indirect dependencies to one `require` block and all the direct one to a different block?

Reply:
Done, moved all direct and indirect dependencies into separate blocks.